### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by ttossavainen**
https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile/issues/15

### CODE REVIEW

File: Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse.csproj [hunk 1]
```diff
@@ -15,9 +15,9 @@
     <PackageTags>Frends</PackageTags>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Description>Task for parsing fixed flat files.</Description>
+    <Description>Frends Task for parsing fixed flat files.</Description>
     <PackageProjectUrl>https://frends.com/</PackageProjectUrl>
-	<RepositoryUrl>https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile/tree/main/Frends.FixedWidthFlatFile.Parse</RepositoryUrl>
+	<RepositoryUrl>https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile</RepositoryUrl>
```
Comment (RikuVirtanen) [inline Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse.csproj]: Why was this changed? Don't we want the correct Task Repository Url to the csproj file?

File: Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/FixedWidthFlatFile.cs
Comment (RikuVirtanen) [inline Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/FixedWidthFlatFile.cs:1]: https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile/pull/16/files#diff-ca469c53aa7ed102d9962a442174cd45b0e16bd591839b8f2e5c469a67621605R8
Please capitalize task.

Comment (RikuVirtanen) [inline Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/FixedWidthFlatFile.cs:1]: https://github.com/FrendsPlatform/Frends.FixedWidthFlatFile/pull/16/files#diff-ca469c53aa7ed102d9962a442174cd45b0e16bd591839b8f2e5c469a67621605L79-L82
Please change to one liner Foreach statement without curly brackets.

**Review by RikuVirtanen — CHANGES_REQUESTED** [review]

File: Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse.csproj (hunk 1 above)
Comment (ttossavainen) [reply Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse.csproj]: Reverted. It was made to match the another Task in this repo.

**Review by RikuVirtanen — APPROVED** [review]
